jenkins/data-prepper/release-data-prepper-all-artifacts.jenkinsfile

```diff
@@ -7,9 +7,8 @@ pipeline {
     agent {
         docker {
             label 'Jenkins-Agent-al2-x64-c54xlarge-Docker-Host'
-            image 'opensearchstaging/ci-runner:ci-runner-ubuntu2004-data-prepper-pipeline-v1'
-            alwaysPull true
-            args '-u root'
+            image 'opensearchstaging/ci-runner:ubuntu2004-x64-docker-buildx0.6.3-qemu5.0-awscli1.22-jdk11-v2'
```

Comment:
The non-Docker stages (e.g. archives and Maven artifacts) need to run on the existing agent (data-prepper-pipeline-v1). 

Please only update the  "Copy Docker Image to DockerHub" and "Copy Docker Image to ECR" stages to use the new agent.

Here is an example:

https://github.com/opensearch-project/opensearch-build/blob/ba7e6cc94bbb9e12a2477c48f1f35791ba791b43/jenkins/docker/docker-copy.jenkinsfile#L33-L36